tests/migration_test.go

```diff
@@ -624,6 +624,38 @@ var _ = Describe("[Serial][rfe_id:393][crit:high][vendor:cnv-qe@redhat.com][leve
 				tests.WaitForVirtualMachineToDisappearWithTimeout(vmi, 240)
 
 			})
+			It("should be able to successfully migrate with a paused vmi", func() {
+				vmi := tests.NewRandomVMIWithEphemeralDisk(cd.ContainerDiskFor(cd.ContainerDiskCirros))
+				tests.AddUserData(vmi, "cloud-init", "#!/bin/bash\necho 'hello'\n")
+
+				By("Starting the VirtualMachineInstance")
+				vmi = runVMIAndExpectLaunch(vmi, 240)
+
+				By("Checking that the VirtualMachineInstance console has expected output")
+				Expect(libnet.WithIPv6(console.LoginToCirros)(vmi)).To(Succeed())
+
+				By("Pausing the VirtualMachineInstance")
+				virtClient.VirtualMachineInstance(vmi.Namespace).Pause(vmi.Name)
+				tests.WaitForVMICondition(virtClient, vmi, v1.VirtualMachineInstancePaused, 30)
+
+				By("verifying that the vmi is still paused after migration")
```

Comment:
should this read
"verifying that the vmi is still paused before migration" instead of "after migration" ?

Reply:
ah, of course